verifiers/internal/gha/verifier.go

```diff
@@ -59,9 +60,11 @@ func verifyEnvAndCert(env *dsse.Envelope,
 		return nil, nil, err
 	}
 
+	fmt.Println("builderID:", builderID.String())
```

Comment:
nit: remove before merging

Reply:
good catch. Completely missed this one. Thanks!